### ISSUE TITLE
Display untrusted extensions at the top of the list 

### ISSUE THREAD

**Issue by FateXBlood** [mihonapp/mihon#66]
### Describe your suggested feature

Once an extension is installed, it is automatically marked as untrusted. In such cases, users with 20+ extensions have to scroll down a lot to mark it as trusted. 

A better idea would be to display the untrusted extensions at the top of the Extension list so that users can mark it as trusted easily without endless scrolling.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an official extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/extensions/issues/new/choose).
- [X] I have updated the app to version **[0.16.0](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by MajorTanya** [comment]
It is slightly easier if you scroll down to the bottom of the untrusted extensions and start with the last. Pretty sure that if you trust one, it will move to the top with all the other trusted ones, and your scroll position would stay at the last untrusted one.

**Comment by FateXBlood** [comment]
@MajorTanya 
That will be the case when one has to mark **all** extensions as trusted. I'm mostly referring to new extensions that a user will install.

**Comment by MajorTanya** [comment]
I see now. While I'd say flinging the list up should get you to the end of the trusted extensions in a second, I understand where you're coming from. Don't know if changing it like that works with what the maintainers have in mind though.

**Comment by suyash01** [comment]
Adding a filter would be a better option then flinging the list as the same page will contain other extensions as well which are part of the repo but not installed.

**Comment by Jaykoerner** [comment]
I am of the personal opinion that trusting extensions is no longer fit for purpose, and that it would be better removed. Instead we should trust repos.


My reasoning for this is that since by default almost all extensions will be untrusted and that given they are more or less standard Android apps (from the operating system's view) almost any malicious or malware concerns are presented from initial install regardless of weather tachiomi forks trust the extension. 


As such since it does not meet a security objective any longer it is an unnecessary feature that just causes extra work for the users

**Comment by AntsyLich** [comment]
Unneeded after #320
